px-map-behavior-layer-IMS.es6.js

```diff
@@ -69,6 +69,18 @@
       },
 
       /**
+       * A property used to determine the zoom level at which the feature becomes visibile
```

Comment:
Probably should change to be plural as it will effect all features in the layer: "...features become visible..."
Also the default value is now `1+` not `1`